cmd/commands/client/command_client.go

```diff
@@ -118,6 +123,13 @@ func (cmd *Command) Start() error {
 	}
 	log.Infof("Api started on: %d", port)
 
+	originalLocation, err := cmd.locationCache.RefreshAndGet()
```

Comment:
Why are't You using/injecting variable `originalLocation` anythere?

Reply:
By the way - I suggest moving log.xx line after location is fetched. In case of error it should display Api started... and then error